tests/Passwordless.Tests/ApplicationEventLogsTests.cs

```diff
@@ -18,7 +18,7 @@ public async Task I_can_view_application_event_logs_when_event_logs_are_enabled(
 
         // Act
         var response = await passwordless.GetEventLogAsync(
-            new GetEventLogRequest { PageNumber = 1, NumberOfResults = 100 }
+            new GetEventLogRequest(1, 100)
```

Comment:
non-blocking-nit: I find this less readable.

Reply:
Same. It's possible to keep the same signature with `required` and `init` but that involves more boilerplate though. We can still do it if it's important.